{"name": "输入错误密码登录失败", "status": "passed", "attachments": [{"name": "stdout", "source": "7753febf-57d6-47e9-a83d-d2f73d23f5f0-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "Parameter", "value": "{'case_no': '5', 'casename': '输入错误密码登录失败', 'mode': '登录', 'data': 'userAccount=979172251@qq.com\\npassword=a1234567', 'assert_way': 'class name=login-form-forgot', 'result': '\\n              Forgot Password?\\n            '}"}], "start": 1651823731748, "stop": 1651823736290, "uuid": "fb36680d-89ea-4c57-8629-a743e58b3ad7", "historyId": "104e3d5feb09afaca2b374c9a621f082", "testCaseId": "f42af1ba93a36a6b78bc4dd2bc5b083c", "fullName": "test_main.Test_Main#test_login", "labels": [{"name": "feature", "value": "登录"}, {"name": "suite", "value": "test_main"}, {"name": "subSuite", "value": "Test_Main"}, {"name": "host", "value": "DESKTOP-NB3FDE0"}, {"name": "thread", "value": "22192-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_main"}]}